tests/data/rolling_stocks/short_slow_rolling_stock.json

```diff
@@ -76,5 +76,6 @@
     }
   },
   "base_power_class": "1",
-  "energy_sources": []
+  "energy_sources": [],
+  "supported_signaling_systems": ["BAL", "BAPR"]
```

Comment:
(I though I already commented but I don't see it, this may cause a duplicate somehow)

I don't think the signaling system should differ from the other example rolling stocks, they generally only differ in what's described in their name. If you want a rolling stock that doesn't support TVM, you can add a new one.

Reply:
done